docs/src/main/asciidoc/_admin-guide/Galleon_provisioning.adoc

```diff
@@ -0,0 +1,217 @@
+[[Galleon_Provisioning]]
+= Provisioning WildFly with Galleon
+
+With the WildFly server being internally built using the https://github.com/wildfly/galleon[Galleon project], WildFly benefits +
+from Galleon's provisioning capabilities. One of the main features that Galleon brings to WildFly is the ability
+to install a complete or customized WildFly server using Galleon tooling.
+
+More information on Galleon features and tooling can be found in the Galleon https://docs.wildfly.org/galleon/[docs].
+
+== WildFly Galleon feature-packs
+
+WildFly is packaged as a Galleon feature-pack maven artifact (a zipped file that contains installation content). 
```

Comment:
s/is packaged as/provides/g

I say that because the standard way people will consume WF is via the traditional zip or tar dist. So our standard packaging is unchanged.